system/Database/MySQLi/Connection.php

```diff
@@ -134,7 +134,7 @@ public function connect(bool $persistent = false)
 					if ($this->encrypt['ssl_verify'])
 					{
 						defined('MYSQLI_OPT_SSL_VERIFY_SERVER_CERT') &&
-						$this->mysqli->options(MYSQLI_OPT_SSL_VERIFY_SERVER_CERT, true);
```

Comment:
Another subset of changes are for using booleans for `bool|int` in places where PHP 8 expects only `int`.